src/__DOCKERFILE_POSTINSTALL_TWEAKS__

```diff
@@ -0,0 +1,2 @@
+# disable sync with udev since the container can not contact udev
+sed -i -e 's/udev_rules = 1/udev_rules = 0/' -e 's/udev_sync = 1/udev_sync = 0/' /etc/lvm/lvm.conf
```

Comment:
I think we should be able to detect if the change got successful. Looks like it's not trivial doing it in sed, maybe greeping after ?

Reply:
done.